client/src/components/performanceForm/performanceForm.tsx

```
@@ -0,0 +1,48 @@
+import { useQuery } from "@tanstack/react-query";
+import Competencies from "./competency";
+import FinalReview from "./finalReview";
+import { KpiScoreTable } from "./kpiTableScore";
+import Api from "@/api/api";
+import { Spinner } from "../ui/spinner";
+import { useAuth } from "@/hooks/useAuthContext";
+import Error from "../Error";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)